moped-toolbox/backfill_complete_srts_ids/.gitignore

```diff
@@ -0,0 +1 @@
+/data/*
```

Comment:
wanna include `secrets.py` here jussssst in case?

Reply:
oh nevermind, i see its covered by the .gitignore one level up

Reply:
i went ahead and changed this to be careful - i definitely had a short moment of panic when I push this branch yesterday 😬 😅